### ISSUE TITLE
How to connect to Mysql ? 

### ISSUE THREAD

**Issue by kevincaradant** [jeedom/documentation#227]
Hi

I tried to connect me but I'm sticking on the connection page.

![capture du 2017-12-22 09-50-52](https://user-images.githubusercontent.com/6093175/34291735-e6ab01d0-e6fd-11e7-8b02-3bddfd99ee0e.png)

To try, I wrote these commands:
```
sudo docker run --name jeedom-mysql -v /opt/jeedom/mysql:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=password -d mysql:latest
```
```
sudo docker run --name jeedom-server --link jeedom-mysql:mysql --privileged -v ~/Documents/jeedom:/var/www/html -e ROOT_PASSWORD=password -p 9080:80 -p 9022:22 jeedom/jeedom
```
Thanks a lot for  your help :)

**Comment by dab0u** [comment]
you should give a try to the forum of the community : https://www.jeedom.com/forum/index.php .
This page is for english users : https://www.jeedom.com/forum/viewforum.php?f=51 .

I hope it'll help you.

**Comment by kevincaradant** [comment]
OK I'm French but maybe my question can be useful for other users :). Anyway, I will look it soon. Thank you.
@dabOu, when you're following the tutorial, line by line, everything works as expected?

**Comment by dab0u** [comment]
Alors va dans la section en français, ça sera bien plus simple ^^ .
Je ne suis pas passé par cette méthode pour l'installation de Jeedom, je ne sais donc pas si ça marche bien ou pas.

**Comment by kevincaradant** [comment]
OK tu n'as pas fait une installation avec Docker si je comprends bien ? Dans ce cas là je vais voir sur le forum si je peux trouver de l'aide

**Comment by dab0u** [comment]
Oui, j'ai directement utilisé un raspberry3, installé un raspbian et ça a très bien fonctionné comme ça.
Le forum va t'apporter beaucoup d'aide, tant que tu sauras expliqué avec détails ton problème :).
D'autant plus si le tuto n'est pas au point, il faut qu'on le sache !

**Comment by kevincaradant** [comment]
Voilà question posée, si cela t’intéresse et que tu souhaites participer:
https://www.jeedom.com/forum/viewtopic.php?f=27&t=32669&p=563228#p563228
